util/ubuntu-setup

```diff
@@ -73,6 +76,34 @@ if [ $UID -ne 0 ]; then
 	exec sudo $0 $@
 fi
 
+# ask for permission to modify apt sources if '-y' flag abesnt
+if [ "ubuntu-18.04" = "$SYSTEM" -a "$YES" != true ]; then
+	echo "Need to add xenial universe to package lists for arm compiler support."
+	read -r -p "Is this okay? [y/N] " response
+	case "$response" in 
+	[yY][eE][sS]|[yY]) 
+        	;;
+    	*)
+		echo "cannot continue on Bionic without permission to add to lists"
+		echo "-- exit"
+		exit 1
+        	;;
+	esac
+fi
+
+if [ "ubuntu-18.04" = "$SYSTEM" ]; then
+	SOURCE_LIST=/etc/apt/sources.list.d/gcc-arm-none-eabi_xenial.list
+	if [ ! -f $SOURCE_LIST ]; then
```

Comment:
Usually when expanding variables we would quote them, to avoid parsing spaces as multiple arguments, like this:

if [ ! -f "$SOURCES_LIST" ]; then

however, in this case, it dosen't matter since the variable is hard coded. I would still prefer this var to be quoted, just in case the path gets a space at some point (and just because it's good practice).